packages/vsce/src/SshConfigUtils.ts

```diff
@@ -482,13 +476,15 @@ export class SshConfigUtils {
 
             config.api.layers.activate(user, global);
 
+            const zoweExplorerApi = ZoweVsCodeExtension.getZoweExplorerApi();
+            const profCache = zoweExplorerApi.getExplorerExtenderApi().getProfilesCache();
             const knownCliConfig: imperative.ICommandProfileTypeConfiguration[] = [
-                ZosUssProfile,
+                // biome-ignore lint/suspicious/noExplicitAny: Accessing protected method
+                ...(profCache as any).getCoreProfileTypes(),
```

Comment:
Should we open an issue to make this method public? 🤔 
Or perhaps we should workaround it with inheritance? 😅